habitat/tasks/nav/nav.py

```diff
@@ -202,6 +203,72 @@ def get_observation(
         )
 
 
+@registry.register_sensor
+class ImageGoalSensor(Sensor):
+    r"""Sensor for ImageGoal observations which are used in ImageGoal Navigation.
+
+    RGBSensor needs to be one of the Simulator sensors.
+    Currently the task is using pointnav Dataset.
```

Comment:
`Currently the task is using pointnav Dataset.` is related to the task, not the sensor. Looks like even with dedicated dataset it may work similar.